packages/istanbul-lib-instrument/package.json

```diff
@@ -14,11 +14,11 @@
     "prepublish": "npm run release"
   },
   "dependencies": {
-    "babel-generator": "^6.18.0",
-    "babel-template": "^6.16.0",
-    "babel-traverse": "^6.18.0",
+    "@babel/generator": "^7.0.0-beta.49",
```

Comment:
I wouldn't use `^` in any of these versions since you never know if we'll make any final breaking changes that may or may not affect this.